core/src/components/toggle/toggle.tsx

```diff
@@ -204,7 +224,15 @@ export class Toggle implements ComponentInterface {
       >
         <div class="toggle-icon" part="track">
           <div class="toggle-icon-wrapper">
-            <div class="toggle-inner" part="handle" />
+            {switchLabelsEnabled && mode === 'ios' && (
+              <ion-icon class="toggle-switch-icon checked" icon={this.getSwitchLabelIcon(true)}></ion-icon>
+            )}
+            <div class="toggle-inner" part="handle">
+              {switchLabelsEnabled && mode === 'md' && this.renderOnOffSwitchLabels()}
```

Comment:
Optional: We can pass in the mode to both `getSwitchLabelIcon` and `renderOnOffSwitchLabels` so we do not need to re-call it again each time the function is called.